ubuntu_bridge_network_configuration.sh

```diff
@@ -0,0 +1,88 @@
+set -xe 
+source lib/logging.sh
+source lib/common.sh
+
+if [ "$MANAGE_PRO_BRIDGE" == "y" ]; then
+     # Adding an IP address in the libvirt definition for this network results in
+     # dnsmasq being run, we don't want that as we have our own dnsmasq, so set
+     # the IP address here
+     IFCFG_PROVISIONING_FILE=/etc/netplan/provision-bridge.yaml
+     IFCFG_PROVISIONING=$(cat << EOF
+network:
```

Comment:
netplan part should be same in any repo .. it is only the renderer that is changed. I suggest make this file as a template and fill the template based on the requirements.

Reply:
Do you get the latest version ? Those lines should not be in `ubuntu_bridge_network_configuration.sh`